server/docker/common-services.yml

```diff
@@ -4,7 +4,6 @@ services:
     volumes:
       - ./../app/:/app/:ro
       - ./../configs:/etc/defold/extender:ro
-      - ./../test-data/sdk:/var/extender/sdk
```

Comment:
Don not mount host folder to every container used for testing. Initial issue was that during parallel execution AuthenticateTest is completed earlier than IntegrationTest. When test completes - defold sdk folder is removed. That leads to fails because IntegrationTests is in progress.